live-examples/css-examples/fonts/font-stretch.html

```diff
@@ -0,0 +1,48 @@
+<section id="example-choice-list" class="example-choice-list" data-property="font-stretch">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">font-stretch: condensed;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: expanded;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: ultra-expanded;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 50%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 100%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+    <div class="example-choice">
+        <pre><code class="language-css">font-stretch: 150%;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+</section>
+
+<div id="output" class="output hidden">
+    <section id="default-example" class="default-example">
+      <div id="example-element">
+          an elephantine lizard
```

Comment:
I think it would be nice for the sample text to be the same as for the [font-variation-settings](https://github.com/mdn/interactive-examples/blob/master/live-examples/css-examples/fonts/font-variation-settings.html#L35) example.

I wondered about using the same font as well, then we couldn't have to add a new one here. But the League Mono seems to be stretchier, and it's the same one as in the body of the page, so perhaps it's a better choice after all.